Dockerfile

```diff
@@ -68,6 +68,16 @@ WORKDIR /home/builder
 USER builder
 ENV PACKAGE=${PACKAGE} ARCH=${ARCH}
 COPY --chown=builder roles/${REPO}.root.json ./rpmbuild/BUILD/root.json
+# We attempt to copy `Licenses.toml` and `licenses` for the current build, otherwise
+# an empty file and a directory are created. Packages that use `bottlerocket-license-tool`
+# will fail the build, since we expect users to provide their own license
```

Comment:
```suggestion
# We attempt to copy `Licenses.toml` and `licenses` for the current build, otherwise
# an empty file and a directory are created so that `bottlerocket-license-tool` will
# fail with a more descriptive error message.
```